### PR TITLE
 zh-CN:site/zh-cn/tutorials/keras/regression.ipynb

### PR DESCRIPTION
**Pull request by nicholaskong**
zh-CN:site/zh-cn/tutorials/keras/regression.ipynb

Create regression.ipynb, and update the latest en/ structure for TF2 docs

Please review the zh-CN:site/zh-cn/tutorials/keras/regression.ipynb

If there are any problems with this translation, please feel free to let me know.

### CODE REVIEW

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 1]
```diff
@@ -0,0 +1,880 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "bBIlTPscrIT9"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/beta/tutorials/keras/basic_regression\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />在 tensorFlow.google.cn 上查看</a>\n",
```
Comment (lamberta) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: Thanks.
Change all `https://www.tensorflow.org/...` domains to `https://tensorflow.google.cn/...`
Here and throughout

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 2]
```diff
@@ -0,0 +1,880 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/keras/basic_regression.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />在 Google Colab 中运行</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/keras/basic_regression.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />在 GitHub 上查看源代码</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/r2/tutorials/keras/basic_regression.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />下载 notebook</a>\n",
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 问题同上，请修改为汉化链接，谢谢！

⬇️ Suggested change
```suggestion
        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/zh-cn/beta/tutorials/keras/basic_regression.ipynb\"><img src=\"https://tensorflow.google.cn/images/download_logo_32px.png\" />下载 notebook</a>\n",
```

**Comment by echosun1996** [comment]
@nicholaskong

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 3]
```diff
@@ -0,0 +1,880 @@
...
+      },
+      "source": [
+        "Note: 我们的 TensorFlow 社区翻译了这些文档。因为社区翻译是尽力而为， 所以无法保证它们是最准确的，并且反映了最新的\n",
+        "[官方英文文档](https://www.tensorflow.org/?hl=en)。如果您有改进此翻译的建议， 请提交 pull request 到\n",
+        "[tensorflow/docs](https://github.com/tensorflow/docs) GitHub 仓库。要志愿地撰写或者审核译文，请加入\n",
+        "[docs-zh-cn@tensorflow.org Google Group](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-zh-cn)。\n",
+        "\n",
+        "在 *回归 (regression)* 问题中，我们的目的是预测出如价格或概率这样连续值的输出。相对于*分类(classification)* 问题，*分类(classification)* 的目的是从一系列的分类出选择出一个分类 （如，给出一张包含苹果或橘子的图片，识别出图片中是哪种水果）。\n",
+        "\n",
+        "本 notebook 使用经典的 [Auto MPG](https://archive.ics.uci.edu/ml/datasets/auto+mpg) 数据集，构建了一个用来预测70年代末到80年代初的汽车验有效率模型。为了做到这一点，我们将为该模型提供许多那个时期的汽车描述。这个描述包含：气缸数，排量数，马力以及重量。\n",
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 修改错别字并调整细节
```suggestion
        "本 notebook 使用经典的 [Auto MPG](https://archive.ics.uci.edu/ml/datasets/auto+mpg) 数据集，构建了一个用来预测70年代末到80年代初汽车燃油效率的模型。为了做到这一点，我们将为该模型提供许多那个时期的汽车描述。这个描述包含：气缸数，排量，马力以及重量。\n",
```

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 4]
```diff
@@ -0,0 +1,880 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "8XKitwaH4v8h"
+      },
+      "source": [
+        "`\"Origin\"` 列是真正的分类，非数字。所以把它转换为独热码 （one-hot）:"
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 我稍作了一些解释。
```suggestion
        "`\"Origin\"` 列实际上代表分类，而不仅仅是一个数字。所以把它转换为独热码 （one-hot）:"
```

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 5]
```diff
@@ -0,0 +1,880 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "Cuym4yvk76vU"
+      },
+      "source": [
+        "### 拆分训练数据集和测试数据集\n",
+        "\n",
+        "现在需要将数据集拆分为一个训练数据集和一个测试数据集。\n",
+        "\n",
+        "我们将使用测试数据集最后对模型进行评估。"
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 调整语序。
```suggestion
        "我们最后将使用测试数据集对模型进行评估。"
```

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 6]
```diff
@@ -0,0 +1,880 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Sj49Og4YGULr"
+      },
+      "source": [
+        "### 检查模型\n",
+        "\n",
+        "使用 `.summary` 方法来打印一个模型的简单描述。"
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 消除歧义。
```suggestion
        "使用 `.summary` 方法来打印该模型的简单描述。"
```

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 7]
```diff
@@ -0,0 +1,880 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "0-qWCsh6DlyH"
+      },
+      "source": [
+        "### 训练模型\n",
+        "\n",
+        "对模型进行1000个时期的训练，并在 `history` 对象中记录训练和验证的准确性。"
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 细节修改。
```suggestion
        "对模型进行1000个周期的训练，并在 `history` 对象中记录训练和验证的准确性。"
```

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 8]
```diff
@@ -0,0 +1,880 @@
...
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "AqsuANc11FYv"
+      },
+      "source": [
+        "该图表显示在约100个 epochs 之后几乎没有改进，或甚至在验证错误中退化。 让我们更新 `model.fit` 调用，当验证值没有提高上是自动停止训练。\n",
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 原文为
This graph shows little improvement, or even degradation in the validation error after about 100 epochs. 
略有机翻的感觉，故稍加修改。
```suggestion
        "该图表显示在约100个 epochs 之后误差非但没有改进，反而出现恶化。 让我们更新 `model.fit` 调用，当验证值没有提高上是自动停止训练。\n",
```

File: site/zh-cn/beta/tutorials/keras/basic_regression.ipynb [hunk 9]
```diff
@@ -0,0 +1,880 @@
...
+        "我们将使用一个 *EarlyStopping callback* 来测试每个 epoch 的训练条件。如果经过一定数量的 epochs 没有显示出改进，则自动停止训练。\n",
```
Comment (echosun1996) [inline site/zh-cn/beta/tutorials/keras/basic_regression.ipynb]: 细节修改。
```suggestion
        "我们将使用一个 *EarlyStopping callback* 来测试每个 epoch 的训练条件。如果经过一定数量的 epochs 后没有改进，则自动停止训练。\n",
```

**Comment by nicholaskong** [comment]
@lamberta @echosun1996 Thanks for your review.
I have submit the latest commit, which include the 'Using https://tensorflow.google.cn/ to instead of https://www.tensorflow.org/ ' and 'Fine-tuning translation'.
Please review again. Thanks a lot!

**Comment by echosun1996** [comment]
I think the translation is excellent. And thanks for volunteering!

**Comment by echosun1996** [comment]
LGTM

**Comment by nicholaskong** [comment]
Abandon this pr.

**Comment by nicholaskong** [comment]
Create regression.ipynb, and update the latest en/ structure for TF2 docs.

Remove the previous basic_regression.ipynb, and push the latest regresion.ipynb to the latest zh-ch path

**Comment by nicholaskong** [comment]
Sorry for multiple commit. I will rebase my fork and pr again

**Pull request closed without merging**